addons/im_livechat/models/chatbot_script_step.py

```diff
@@ -367,25 +367,31 @@ def _process_step_forward_operator(self, discuss_channel, users=None):
                 posted_message = discuss_channel._chatbot_post_message(self.chatbot_script_id, plaintext2html(self.message))
 
             # next, add the human_operator to the channel and post a "Operator joined the channel" notification
-            discuss_channel.with_user(human_operator).sudo().add_members(human_operator.partner_id.ids, open_chat_window=True)
+            discuss_channel.with_user(human_operator).sudo().add_members(
+                human_operator.partner_id.ids
+            )
             # sudo - discuss.channel: let the chat bot proceed to the forward step (change channel operator, add human operator
             # as member, remove bot from channel, rename channel and finally broadcast the channel to the new operator).
             channel_sudo = discuss_channel.sudo()
-            channel_sudo.livechat_operator_id = human_operator.partner_id
             if bot_member := channel_sudo.channel_member_ids.filtered(
                 lambda m: m.partner_id == self.chatbot_script_id.operator_partner_id
             ):
                 channel_sudo._action_unfollow(partner=bot_member.partner_id)
             # finally, rename the channel to include the operator's name
-            channel_sudo.name = " ".join(
```

Comment:
Note: moving to single write to do a single bus send for both name and operator (name was already sent operator added in this PR)